paimon-core/src/test/java/org/apache/paimon/utils/TagManagerTest.java

```diff
@@ -136,6 +138,66 @@ public void testCreateTagWithTimeRetained() throws Exception {
         assertThat(tags.get(0).getValue()).contains("tag");
     }
 
+    @Test
+    public void testCreateTagWithMultiThreads() throws Exception {
```

Comment:
This method can not be failed in the master.

Reply:
I've run it many times at my machine.

Reply:
https://github.com/apache/paimon/blob/0bad12d4d04a6fb52bae2203964d7e70365a76fb/paimon-core/src/main/java/org/apache/paimon/utils/TagManager.java#L137

It used `overwriteFileUtf8` which will overwrite the tag file if some other procedure is creating the tag. And it won't throw exceptions. I think it's not safe?